docs/flink-connector.md

```diff
@@ -35,10 +35,8 @@ To create the table in Flink SQL by using SQL syntax `CREATE TABLE test (..) WIT
 
 * `connector`: Use the constant `iceberg`.
 * `catalog-name`: User-specified catalog name. It's required because the connector don't have any default value.
-* `catalog-type`: Default to use `hive` if don't specify any value. The optional values are:
-    * `hive`: The Hive metastore catalog.
-    * `hadoop`: The hadoop catalog.
-    * `custom`: The customized catalog, see [custom catalog](../custom-catalog) for more details.
+* `catalog-type`: `hive` or `hadoop` for built-in catalogs (defaults to `hive`), or left unset for custom catalog implementations using `catalog-impl`.
+* `catalog-impl`: The fully-qualified class name custom catalog implementation, must be set if `catalog-type` is unset. See also [custom catalog](../flink/flink-getting-started.md#custom-catalog) for more details.
```

Comment:
I think `catalog-impl` is an optional and most of the time users don't need to set it. Should we indicate that this configuration is optional and indicate the default value?

Reply:
I think it should be clear from **must be set if `catalog-type` is unset**  that this is optional

Reply:
I agree it should be clear already that one or the other needs to be set from the language already.